app/graphql/mutations/create_order.rb

```diff
@@ -1,11 +1,16 @@
 module Types
+    class CartItemInputType < Types::BaseInputObject
+        argument :product_id, ID, required: true
+        argument :quantity, Int, required: true
+    end
+
     class CreateOrderInputType < Types::BaseInputObject
         argument :billing_customer_id, Int, required: true
         argument :shipping_customer_id, Int, required: true
         argument :shipping_cost, Int, required: false
         argument :tax_cost, Int, required: true
         argument :unit_price, Int, required: true
-        argument :cart, [Types::CartItemType], required: true
+        argument :cart, [Types::CartItemInputType], required: true
```

Comment:
GraphQL was expecting an `InputType` that inherits from `Types::BaseInputObject`, so I had to create this new type above